### DISCUSSION TITLE
Page elements wrapping on mobile recently

### DISCUSSION THREAD

**Discussion by hamiltlr** [firefly-iii/firefly-iii#8214] [General]
I've noticed recently that page elements near the right side have started wrapping to the next line, causing layout issues. I'm using Chrome on Android 14. Here's a couple screenshots of the demo site:

![Screenshot_20231128-132001.png](https://github.com/firefly-iii/firefly-iii/assets/1261920/8aa5f0dc-cc45-48ca-a51b-361e3da6f2f5)

![Screenshot_20231128-132010.png](https://github.com/firefly-iii/firefly-iii/assets/1261920/d081490c-d926-4209-b5e4-5d5c7e91e3de)

I am visually impaired so it's possible I have some display setting set to larger than normal, but I don't remember changing anything recently and it used to look fine. I also see the same results on Firefox. 

Anyone else noticing this?

**Comment by JC5** [comment]
Mmm, I haven't seen this myself and the layout hasn't changed in the past few months. Perhaps it's simply the long month name?

**Reply by hamiltlr** [reply]
Interesting. The month name does seem to affect it. Changing to August moves the "+" icon back up, although it still overlaps slightly on the new transaction pane

![Screenshot_20231202-093134.png](https://github.com/firefly-iii/firefly-iii/assets/1261920/50eeb785-cfd3-4806-a774-4cef8235b4ac)

Although I honestly can't remember if that has always happened or not.
